### ISSUE TITLE
ignore whole rules

### ISSUE THREAD

**Issue by stephanfriedrich** [yargalot/AccessSniff#17]
hi, thx for nice your tool, i use it with "grunt-accessibility". 

**Question:**
is it possible, to ignore whole rules instead of specify each rule.

example:

```
"ignore": [
    "WCAG2AA.Principle4.Guideline4_1.4_1_1.F77",
    "WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail",
     "WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail",
     "WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Abs",
     "WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.BgImage"
  ]
```

could be:

```
"ignore": [
    "WCAG2AA.Principle4.Guideline4_1.4_1_1.F77",
    "WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail",
     "WCAG2AA.Principle1.Guideline1_4.1_4_3.G18"    
  ]
```

**Comment by yargalot** [comment]
Done in 3.0
